docs/git-command-table.yml

```diff
@@ -106,6 +106,13 @@
     `rm filename`
   Notes: ''
 
+- Use case: Remove a previously tracked file from the current change, but keep it in the working copy
+  Git command: >
+    `git rm --cached filename`
+  Jujutsu command: >
+    `jj file untrack filename`
+  Notes: ''
```

Comment:
I think we should have a note that the file must be added to `.gitignore` or `.git/info/exclude` or else it will be automatically tracked again. Maybe something like this: "File name must match an ignore pattern to remain untracked."

Maybe there's a good doc elsewhere we can link to for more info? I'm not sure if this command table supports links.

Reply:
Not sure if I got the link part right, but I added your comment to the file. I'm assuming that the `<a href=...>` will end up in the final page at some point